CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Change Log
 
+## 3.0.0
```

Comment:
We probably want to release this PR with the path-to-regex package update as well.